src/Extensions/Logging/Internal/Replayable/ReplayableActivity.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Logging.Replayable
+{
+	internal class ReplayableActivity : Activity
+	{
+		public ReplayableActivity(string operationName, uint maxReplayedEventsPerActivity) : base(operationName)
```

Comment:
`uint` is not a CLS type, so this can't necessarily be used by other .NET languages. It's not necessarily a problem here as it is internal, but I'm letting you know in case this extends to the public interface somewhere I've missed.

Reply:
It's public in `OmexLoggingOptions` but tradeoff here is to either:
1. Use `uint` and make sure that there will be no negative values (maybe we could event use `ushort`) and risk that won't be used in some other language
2. Use `int` to support other languages for theoretical support of languages that we don't use?
Do you think it worth it?